tests/Search20Test.php

```diff
@@ -284,13 +284,24 @@ public function testSearchByCustomLanguageTag2()
 
     public function testSearchByPopularity()
     {
-      $json = $this->s->GetSearchMatches(null, 'p:*', 1, 1);
+      $json = $this->s->GetSearchMatches(null, 'p:popularity', 1, 1);
```

Comment:
Do you want to keep a test case that `p:*` searches by popularity?

And `p:!` searches alphabetically?